Sources_v3/Query/Filter.swift

```diff
@@ -62,6 +62,11 @@ public enum Filter: Encodable, CustomStringConvertible {
     /// Filter without any of filters (like `not or`).
     indirect case nor([Filter])
     
+    // MARK: - Technical
+
+    // Add comment!
+    indirect case explicitFilterHash(Filter, String)
+    
```

Comment:
Do we still need this? Maybe we can just change `ChannelUpdater.update` method to accept optional explicit filter hash?

Reply:
It was first idea, but in that case we will have to modify 3 functions. 
It seems like this technical enum solves this issue nicer.

Reply:
All right then 👍